### PR TITLE
[O2B-421] Trigger notification on log creation

### PR DESCRIPTION
**Pull request by awegrzyn**

### CODE REVIEW

File: lib/config/notification.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+// Default configuration
+const brokers = [process.env?.NOTIFICATION_BROKER] || null;
+
+module.exports = {
+    brokers,
```
Comment (graduta) [inline lib/config/notification.js:18]: I believe the notification service from web-ui expects an array;
Could you confirm that the process.env will return an `Array` and not a `String`?

File: lib/server/controllers/logs.controller.js [hunk 2]
```diff
@@ -54,6 +58,11 @@ const createLog = async (request, response, next) => {
         response.status(Number(error.status)).json({ errors: [error] });
     } else {
         response.status(201).json({ data: result });
+        const url = `${request.protocol}://${request.get('host')}?page=log-detail&id=${result.id}`;
+        const tags = result.tags.map((tag) => tag.text);
+        if (notification && notification.isConfigured()) {
```
Comment (graduta) [inline lib/server/controllers/logs.controller.js:63]: Here you could use the same syntax you used above
```suggestion
        if (notification?.isConfigured()) {
```

**Review by graduta — CHANGES_REQUESTED** [review]

File: lib/config/notification.js (hunk 1 above)
Comment (awegrzyn) [reply lib/config/notification.js:18]: Nope, it will not, but that's why `[` and `]` were added.

Comment (graduta) [reply lib/config/notification.js:18]: Ah, ok. I missed the brackets as I was thinking we are expecting multiple brokers separated by commas

**Review by graduta — DISMISSED** [review]

**Review by graduta — APPROVED** [review]

**Pull request merged**